icons/warzone2100.desktop

```diff
@@ -1,5 +1,5 @@
 [Desktop Entry]
-Version=1.0
+Version=1.4
```

Comment:
@volkov0aa:

According to:
https://specifications.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html
> Version of the Desktop Entry Specification that the desktop entry conforms with. Entries that confirm with this version of the specification should use `1.1`. Note that the version field is not required to be present.

It's not clear to me if that's a missing (outdated) reference, or if the Version field should indeed be 1.1 (even though the [Changes to this Specification](https://specifications.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html#idm45233126766896) section specifies that `PrefersNonDefaultGPU` was added in 1.4).

Reply:
Most likely they forgot to update version number in the description of Version key.
But the spec also states,  that "the version field is not required to be present".
Also desktop-file-validate doesn't check whether presented keys correspond to Version, it just checks that Version is correct: https://github.com/freedesktop/xdg-desktop-file-utils/blob/master/src/validate.c#L961
I don't think this key is of any importance.

Reply:
Anyway, here is the merge request to fix desktop entry spec: https://gitlab.freedesktop.org/xdg/xdg-specs/-/merge_requests/39